index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Calculator Benchmark</title>
+  <meta charset="utf-8">
+  <meta name="description" content="calculator">
+  <meta name="author" content="Keith Oppel">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:100" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400" rel="stylesheet">
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.css" rel="stylesheet">
+  <link type="text/css" rel="stylesheet" href="main.css">
+</head>
+
+<body>
+  <div class="calculator">
+    <div class="calculator-display">
+      <span class="display">0</span>
+    </div>
+    <div class="calculator-row">
+      <div class="calculator-button"><button value="AC">AC</button></div>
```

Comment:
One of things you could do make it easier to read is to put spaces in between the  closed and open button tag. One example is line 21 the AC, well for me, was very hard to detect. I just learned something out of your goals...I didn't know that you can create a div within a div. Thanks!